src/Question.vue

```diff
@@ -3,53 +3,72 @@
         <div class="body-wrapper">
             <!-- Default slot is question body -->
             <slot></slot>
-            <div v-if="hasInput" class="textarea-container">
+            <div v-if="hasInputBool" class="textarea-container">
                 <textarea class="form-control question-input" rows="3" placeholder="write your answer here..."></textarea>
             </div>
         </div>
-        <accordion>
-            <panel v-show="hasHintSlot" header="Hint" expandable no-close>
-                <slot name="hint">
-                    No hint is available for this question.
-                </slot>
-            </panel>
-            <panel v-show="hasAnswerSlot" header="Answer" expandable no-close>
-                <slot name="answer"></slot>
-            </panel>
-        </accordion>
+        <panel v-show="hasHintSlot" header="Hint" expandable no-close>
+            <template v-if="isEmptyHint">
```

Comment:
Can use `div` instead of `template`. Same for the one below and for the answer slot too.

Reply:
Using `<div>` for **v-if** will cause a redundant `<div>` tag to be rendered. Template doesn't have any excess html code.